website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,74 @@
+
+When a feature or function is conditionally available depending on other steps that must be taken, provide context in place of that action so that users are able to take those steps to enable this feature.
+
+!!! Dont
+
+In this example, the button is disabled and no context is provided, leaving the user confused and unaware of the steps required to enable the feature.
+
+![Showing an interface with a disabled button that doesn't describe why that button is disabled. This is not a recommended experience.](/assets/patterns/disabled-patterns/conditional-availability-example-disabled.png)
+
+Disabled elements lack interactive states, preventing a user from hovering, focusing, or clicking them. For users with assistive technology (e.g., a screen reader), disabled elements are almost entirely ignored, including the text within.
+
+!!!
+
+
+!!! Do
+
+Instead, we recommended replacing the disabled element with contextual guidance to help users to understand why this feature isn't available, and providing them with clear steps to enable it.
+
+![Showing an interface with a compact alert explaining why the expected behavior isn't there with guidance.](/assets/patterns/disabled-patterns/conditional-availability-example-explanation.png)
+
+If possible, provide CTAs directly to the place where users can take immediate action.
+
+!!!
+
+
+### Upgrading
+
+When premium features are available and a user has the necessary permissions, highlighting a path to upgrade provides a graceful marketing opportunity.
+
+An example of this is Figma's "create branch" function that is only available at the enterprise tier. Clicking "create a branch" in Figma opens a marketing modal that entices users to upgrade to gain access to this feature.
+
+![Screen shot of figma's modal showing information about how to upgrade to enterprise to gain access to creating branches.](/assets/patterns/disabled-patterns/upgrade-modal-example.png)
+
+
+### Service outage
+
+In instances where outages may occur, provide a clear message to users on what they can do and add a link to [our status page](https://status.hashicorp.com/).
+
+An example of this is Instagram's login page. When a user has no connection to their servers, a message will appear on submit.
+
+![Displaying Instagram's login page with an error below notifying the user they have no connection to instagram.](/assets/patterns/disabled-patterns/service-outage-example.png)
+
+### Incomplete flow
+
+In a form, regardless of whether all of the fields have been filled out, always enable the submit button. When we disable the only way to move forward, users are often left confused and frustrated. Instead, allow users to understand what erorrs took place so that they can fix them and proceed in the form's completion.
+
+In this example, creating a cluster has required inputs. Clicking "create cluster" will display errors where applicable, allowing the user to understand where the problems are to remediate them.
```

Comment:
[suggestion] This uses a bit more plain language

```suggestion
In this example, creating a cluster has required inputs. Clicking "create cluster" will display any errors, informing the user of errors and how to fix them.
```

Reply:
Also, I'm not sure if this got lost somewhere (I don't see the comment anymore), but this paragraph references the entire form (click create cluster, other fields as requirements), but only shows one field. I thought I had left a previous note about considering adding more of the UI around this example.

Reply:
Yeah, ill add more UI. Thanks for the suggestion :)

Reply:
Updated UI and text to reflect the addition in the image.